tests/test_filter_by.py

```diff
@@ -31,9 +34,21 @@ def test_filter_by(self):
         self.assertEqual(2, len(users))
 
         users = self.loop.run_until_complete(TestUser.filter_by(self.db, status='active'))
-        self.assertEqual(9, len(users))
-
-
+        self.assertEqual(10, len(users))
 
+    def test_filter_by(self):
```

Comment:
This redefines a previous `test_filter_by` method in this same file (scroll up), so only one of the two tests are being run. This would normally throw a flake8 error, but apparently we're not running flake8 on this repo. Can you add that?